src/TLuaInterpreter.h

```diff
@@ -568,6 +568,10 @@ class TLuaInterpreter : public QThread
     static int unzipAsync(lua_State* L);
     static int setMapWindowTitle(lua_State*);
     static int getMudletInfo(lua_State*);
+    static int getMapBackgroundColor(lua_State*);
+    static int setMapBackgroundColor(lua_State*);
+    static int getMapRoomExitsColor(lua_State*);
```

Comment:
Can't just `setMapExitsColor` work?

Reply:
Well, I wouldn't want to get them confused with Map *Area* Exits - which also have a special presentation in both 2D and 3D maps.